### PR TITLE
 Admin command to manage pod network

### PR DESCRIPTION
**Pull request by pravisankar**
Updated SDN architecture:
- Added master and node SDN configuration
- Migration between network plugins

### CODE REVIEW

**Comment by pravisankar** [comment]
origin PR: https://github.com/openshift/origin/pull/5024

**Comment by pravisankar** [comment]
cc: @adellape

**Comment by adellape** [comment]
@pravisankar Needs to go in the build_cfg spec for it to show up in the nav.

For now you can just throw it in admin_guide, but (and I could do this in a follow-up because it would require a lot of link fixing) do you think we have enough networking/routing/HA-related topics in the Admin Guide at this point that we could start a "Networking" (or some similar name) subcategory? Something like:

```
Name: Administration
Dir: admin_guide
Topics:
  - Name: Overview
    File: index
...
  - Name: Networking
    Dir: networking
    Topics:
      - Name: Overview
        File: index
      - Name: IPtables
        File: iptables
      - Name: Working with HTTP Proxies
        File: http_proxies
      - Name: Native Container Routing
        File: native_container_routing
      - Name: Routing from Edge Load Balancers
        File: routing_from_edge_lb
      - Name: Managing Pod Networks
        File: pod_network
      - Name: High Availability
        File: high_availability
      - Name: Monitoring Routers
        File: router
```

File: architecture/additional_concepts/sdn.adoc [hunk 1]
```diff
@@ -27,17 +27,17 @@ OpenShift SDN provides two SDN plug-ins for configuring the network:
...
-project. Pods from different projects cannot send packets to or receive
-packets from pods and services of a different project.
+* The *multitenant* plug-in provides OpenShift project level isolation for
+pods and services. Each project receives a unique Virtual Network ID (VNID)
+that identifies traffic from pods assigned to the project. Pods from different
+projects cannot send packets to or receive packets from pods and services of
+a different project.
 +
-However, the *default* project (which receives VNID 0) is more privileged in
+However, projects which receives VNID 0 is more privileged in
```
Comment (adellape) [inline architecture/additional_concepts/sdn.adoc]: s/is/are/

Comment (adellape) [inline architecture/additional_concepts/sdn.adoc]: s/receives/receive/

File: architecture/additional_concepts/sdn.adoc [hunk 2]
```diff
@@ -27,17 +27,17 @@ OpenShift SDN provides two SDN plug-ins for configuring the network:
...
 that it is allowed to communicate with all other pods, and all other pods can
-communicate with the *default* project. This facilitates certain services (like
-the load balancer) that are run in containers themselves and must talk to all
-other containers.
+communicate with this project. In OpenShift cluster, *default* project has
```
Comment (adellape) [inline architecture/additional_concepts/sdn.adoc]: s/this project/them/ (maybe?)
s/cluster/clusters/
s/*default*/the *default*/

File: architecture/additional_concepts/sdn.adoc [hunk 3]
```diff
@@ -27,17 +27,17 @@ OpenShift SDN provides two SDN plug-ins for configuring the network:
...
-packets from pods and services of a different project.
+* The *multitenant* plug-in provides OpenShift project level isolation for
+pods and services. Each project receives a unique Virtual Network ID (VNID)
+that identifies traffic from pods assigned to the project. Pods from different
+projects cannot send packets to or receive packets from pods and services of
+a different project.
 +
-However, the *default* project (which receives VNID 0) is more privileged in
+However, projects which receives VNID 0 is more privileged in
 that it is allowed to communicate with all other pods, and all other pods can
```
Comment (adellape) [inline architecture/additional_concepts/sdn.adoc]: s/it is/they are/

**Comment by adellape** [comment]
@pravisankar Some nits inline, but LGTM in general once it's added to the Admin Guide in the build_cfg. Thanks!

**Comment by pravisankar** [comment]
Updated

**Comment by vikram-redhat** [comment]
This was followed up here: https://github.com/openshift/openshift-docs/pull/2766. Removing the to_followup.

**Pull request merged**